src/lib/cookie/cookie.js

```diff
@@ -98,6 +119,16 @@ function encodeVendorConsentData(vendorData) {
 	return noRangesData.length < rangesData.length ? noRangesData : rangesData;
 }
 
+
+function decodeBitsToIds(bitString) {
```

Comment:
This function is used only in `decodeVendorConsentData` which is no longer used

Reply:
I've moved it to separate file which is bundled only for docs.